### ISSUE TITLE
Can't save Drafts or Send e-mail issues

### ISSUE THREAD

**Issue by Danzign** [RainLoop/rainloop-webmail#2197]
**RainLoop version, browser, OS: 1.17, Chrome and Firefox

**Expected behavior and actual behavior: To send e-mail and save drafts

**Steps to reproduce the problem: Typing e-mail, then try to save or send e-mail got error. If e-mail contain this string Data: I go the error, as you can see on the Screenshots

**Logs or screenshots:**
![WhatsApp Image 2022-11-22 at 16 01 11](https://user-images.githubusercontent.com/119603590/205071973-9ad2a87b-32a9-446b-9220-d01197ece23d.jpeg)
![WhatsApp Image 2022-11-23 at 13 58 01](https://user-images.githubusercontent.com/119603590/205071979-6e815d0c-6dcc-45ad-b1fa-9ea005c309ff.jpeg)

**Comment by homegrownhuman** [comment]
Hey Danzign, have you been able to solve the issue? I Have had the same for a while and I can´t decide what could have caused it. I would love to hear from you!

**Comment by Danzign** [comment]
No, I just stopped using the tool when I read in some comments around here that it could be discontinued. Didn't want to risk the service.
